docs/README.rst

```diff
@@ -0,0 +1,204 @@
+Schematic
```

Comment:
I'm not a fan that this `README.rst` file contains duplicated content from the main `README.md` file. It's only a matter of time until they diverge in confusing ways. If we're moving to using ReadTheDocs, we should perhaps replace the content in `README.md` with a link to the documentation page.

Reply:
Yup, you're right. We can link different sections of the GitHub README to the documentation page(s). We can think/discuss how we need to re-write the README docs.